src/groupeddataframe/splitapplycombine.jl

```diff
@@ -1447,187 +1440,57 @@ function _combine_tables_with_first!(first::Union{AbstractDataFrame,
 end
 
 """
-    by(d::AbstractDataFrame, cols::Any, args...;
-       sort::Bool=false, skipmissing::Bool=false, keepkeys::Bool=true)
-    by(fun::Union{Function, Type}, d::AbstractDataFrame, cols::Any;
-       sort::Bool=false, skipmissing::Bool=false, keepkeys::Bool=true)
-    by(pair::Pair, d::AbstractDataFrame, cols::Any;
-       sort::Bool=false, skipmissing::Bool=false, keepkeys::Bool=true)
-    by(d::AbstractDataFrame, cols::Any, fun::Union{Function, Type};
-       sort::Bool=false, skipmissing::Bool=false, keepkeys::Bool=true)
-    by(d::AbstractDataFrame, cols::Any, pair::Pair;
-       sort::Bool=false, skipmissing::Bool=false, keepkeys::Bool=true)
-
-Split-apply-combine in one step: apply `fun`, `pair` or `args` to each grouping
-in `df` based on grouping columns `cols`, and return a `DataFrame`.
-This is a shorthand for `combine` called on
-`groupby(df, cols, sort=sort, skipmissing=skipmissing)`.
-
-$F_ARGUMENT_RULES
-
-$F_TYPE_RULES
-
-$KWARG_PROCESSING_RULES
-
-The resulting data frame will be sorted if `sort=true` is passed.
-Otherwise, ordering of rows is undefined.
-
-If `skipmissing=true` rows with `missing` values in one of the grouping columns
-`cols` will be skipped.
-
-See [`groupby`](@ref) and [`combine`](@ref) and for details and more examples.
-
-# Examples
-```jldoctest
-julia> df = DataFrame(a = repeat([1, 2, 3, 4], outer=[2]),
-                      b = repeat([2, 1], outer=[4]),
-                      c = 1:8);
-
-julia> by(df, :a, :c => sum, nrow)
-4×3 DataFrame
-│ Row │ a     │ c_sum │ nrow  │
-│     │ Int64 │ Int64 │ Int64 │
-├─────┼───────┼───────┼───────┤
-│ 1   │ 1     │ 6     │ 2     │
-│ 2   │ 2     │ 8     │ 2     │
-│ 3   │ 3     │ 10    │ 2     │
-│ 4   │ 4     │ 12    │ 2     │
-
-julia> by(sdf -> sum(sdf.c), df, :a) # Slower variant
-4×2 DataFrame
-│ Row │ a     │ x1    │
-│     │ Int64 │ Int64 │
-├─────┼───────┼───────┤
-│ 1   │ 1     │ 6     │
-│ 2   │ 2     │ 8     │
-│ 3   │ 3     │ 10    │
-│ 4   │ 4     │ 12    │
-
-julia> by(df, :a) do d # do syntax for the slower variant
-           sum(d.c)
-       end
-4×2 DataFrame
-│ Row │ a     │ x1    │
-│     │ Int64 │ Int64 │
-├─────┼───────┼───────┤
-│ 1   │ 1     │ 6     │
-│ 2   │ 2     │ 8     │
-│ 3   │ 3     │ 10    │
-│ 4   │ 4     │ 12    │
+    select(gd::GroupedDataFrame, args...;
+           copycols::Bool=true, keepkeys::Bool=true, regroup::Bool=false)
 
-julia> by(df, :a, :c => (x -> sum(log, x)) => :sum_log_c) # specifying a name for target column
-4×2 DataFrame
-│ Row │ a     │ sum_log_c │
-│     │ Int64 │ Float64   │
-├─────┼───────┼───────────┤
-│ 1   │ 1     │ 1.60944   │
-│ 2   │ 2     │ 2.48491   │
-│ 3   │ 3     │ 3.04452   │
-│ 4   │ 4     │ 3.46574   │
+Apply `args` to `gd` following the rules described in [`combine`](@ref).
+Ensure that the return value has number of rows equal to `nrow(parent(gd))`.
```

Comment:
That doesn't sound correct, as broadcasting happens contrary to `combine`.

Reply:
`combine` also broadcasts (just to any number of rows).

I have updated the docs + added examples + added `See also` sections everywhere.